src/main/kotlin/com/openlattice/mechanic/upgrades/AddPgAuditToExistingOrgs.kt

```diff
@@ -0,0 +1,54 @@
+package mechanic.src.main.kotlin.com.openlattice.mechanic.upgrades
+
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.mechanic.Toolbox
+import com.openlattice.mechanic.upgrades.Upgrade
+import com.openlattice.mechanic.upgrades.Version
+import com.openlattice.organizations.Organization
+import com.openlattice.postgres.external.ExternalDatabaseConnectionManager
+import org.slf4j.LoggerFactory
+
+/**
+ * @author Andrew Carter andrew@openlattice.com
+ */
+
+class AddPgAuditToExistingOrgs(
+    private val toolbox: Toolbox,
+    private val extDbConnMan: ExternalDatabaseConnectionManager
+
+) : Upgrade {
+
+    companion object {
+        private val logger = LoggerFactory.getLogger(AddPgAuditToExistingOrgs::class.java)
+        private const val PGAUDIT_EXTENSION = "pgaudit"
+        private const val OPENLATTICE_SCHEMA = "openlattice"
+    }
+
+    override fun upgrade(): Boolean {
+        val allOrgs = HazelcastMap.ORGANIZATIONS.getMap(toolbox.hazelcast).values.toList()
+
+        allOrgs.forEach { addPgAuditToOrg(it) }
+        return true
+    }
+
+    private fun addPgAuditToOrg(org: Organization) {
+        logger.info("Adding pgaudit for org: ${org.title} [${org.id}]")
+
+        extDbConnMan.connectToOrg(org.id).let { hds ->
+            hds.connection.use { connection ->
+                connection.createStatement().use { stmt ->
+
+                    // create pgaudit extension
+                    stmt.execute(CREATE_EXTENSION_SQL)
+                }
+            }
+        }
+        logger.info("Finished adding pgaudit extension for org: ${org.title}")
+    }
+
+    private val CREATE_EXTENSION_SQL = "CREATE EXTENSION IF NOT EXISTS $PGAUDIT_EXTENSION SCHEMA $OPENLATTICE_SCHEMA"
```

Comment:
No change required, just want to confirm ... there's no need for wrapping `$PGAUDIT_EXTENSION` and `$OPENLATTICE_SCHEMA`, right?

Reply:
Correct, `$PGAUDIT_EXTENSION` and `$OPENLATTICE_SCHEMA` are constants declared in the class's companion object so there is no functional need to wrap them.